04_numeronym/jimbotech/numeronym_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+type numeronymTest struct {
+	input    string
+	expected string
+}
+
+var numTests = []numeronymTest{
```

Comment:
Move this into the function that uses it. No need to be global scope. It makes the test self-contained and easier to read without having to go back and forth.

You can also remove the `numeronymTest` struct - it really isn't needed:
```
func TestWordMixer(t *testing.T) {
    tests := []struct{
        input, expected string
    }{
        {"Kubernetes, "K8s"},
        ...
    }
    ...
}
```

See: https://talks.golang.org/2012/10things.slide#3 (and the rest of that slide deck while you're there)